cunumeric/fft/fft.py

```diff
@@ -646,7 +648,7 @@ def rfftn(
             return r2c.fft(
                 s=s[0:-1],
                 axes=axes[0:-1],
-                kind=fft_type.complex,
+                kind=cast(FFTType, fft_type.complex),
```

Comment:
not sure how to avoid this cast without some code changes.

Reply:
Fixed by strengthening a method signature in `FFTType` 594e492086b03e78d676d525b8052a4f34e131dc